src/App.js

```diff
@@ -13,39 +13,34 @@ const App = () => {
   const [data, setData] = useState(StorageData.GetLocal() || []);
 
   const handelClick = () => {
-    if (!firstName || !LastName || !age)
-      Swal.fire({
-        title: 'Error!',
-        text: 'Check data please',
-        icon: 'error',
-        confirmButtonText: 'OK',
-      });
-    else {
+    if (!firstName || !LastName || !age) {
+      SweetAlert.GetAlert('Check data please', 'OK', 'Error!');
```

Comment:
the second argument should be boolean and why you send it instead of add them as default ?